addons/sale/views/sale_portal_templates.xml

```diff
@@ -486,9 +488,9 @@
                         <div t-field="sale_order.partner_shipping_id" t-options='{ "widget": "contact", "fields": [ "name", "address"]}'/>
                     </span>
                 </div>
-                <div id="sale_invoices" class="col-12 col-lg-5 mb-3 mb-lg-0">
+                <div id="sale_invoices" t-if="invoices and state in ['sale', 'cancel']">
```

Comment:
`invoices` is defined on the following line, invoice details are not shown anymore after your changes.